js/pages/questionnaire.js

```diff
@@ -347,6 +367,23 @@ export const blockParticipant = () => {
 
 }
 
+const buildSoccerResults = async (title, task) => {
+
+    let soccerURL = '';
```

Comment:
We can save one line by doing ` let soccerURL = SOCcerDev;`, and change its value for stage and prod cases.